src/mgmt/svc.pb-c.c

```diff
@@ -1444,17 +1522,43 @@
     0,             /* flags */
     0,NULL,NULL    /* reserved1,reserved2, etc */
   },
+  {
+    "secondary_rank_uris",
+    5,
+    PROTOBUF_C_LABEL_REPEATED,
+    PROTOBUF_C_TYPE_MESSAGE,
+    offsetof(Mgmt__GetAttachInfoResp, n_secondary_rank_uris),
+    offsetof(Mgmt__GetAttachInfoResp, secondary_rank_uris),
+    &mgmt__get_attach_info_resp__rank_uri__descriptor,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
+  },
+  {
+    "secondary_client_net_hints",
+    6,
+    PROTOBUF_C_LABEL_REPEATED,
+    PROTOBUF_C_TYPE_MESSAGE,
+    offsetof(Mgmt__GetAttachInfoResp, n_secondary_client_net_hints),
+    offsetof(Mgmt__GetAttachInfoResp, secondary_client_net_hints),
+    &mgmt__client_net_hint__descriptor,
+    NULL,
+    0,             /* flags */
+    0,NULL,NULL    /* reserved1,reserved2, etc */
```

Comment:
(style)  space required after that ',' (ctx:VxV)
(style)  space required after that ',' (ctx:VxV)